### PR TITLE
Fix the problem on an attempt to destroyEditor while it is not ready

### PR DESCRIPTION
**Pull request by VLZH**

### CODE REVIEW

**Comment by VLZH** [comment]
Sorry... i will add a one more commit

**Comment by VLZH** [comment]
So... Problem in editor.js. Problem with a situation when a holder is deleted and we try to destroy editor instance. I think that this PR is correct.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,9 @@
+# environment and tools
```
Comment (Jungwoo-An) [inline .gitignore]: I don't think this change is necessary. sorry..!

File: lib/EditorJs.tsx [hunk 2]
```diff
@@ -55,12 +55,14 @@ class EditorJsContainer extends React.PureComponent<Props> {
   }
 
   destroyEditor () {
-    if (!this.instance) {
-      return
+    if (this.instance) {
+      this.instance.isReady.then(() => {
+        if (this.instance) {
+          this.instance.destroy()
+          this.instance = undefined
```
Comment (Jungwoo-An) [inline lib/EditorJs.tsx]: 👍awesome!

File: .gitignore (hunk 1 above)
Comment (VLZH) [reply .gitignore]: Okey👍 Do you mean the file or some strings in it?

Comment (Jungwoo-An) [reply .gitignore]: it means .gitignore 👍

Comment (VLZH) [reply .gitignore]: Ok) I removed one commit from PR

**Comment by Jungwoo-An** [comment]
@VLZH this pr is merged! Thanks!! 👍

**Pull request merged**